public/index.html

```diff
@@ -38,7 +38,7 @@
     </script>
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/FortAwesome/Font-Awesome@5.13.0/css/all.min.css">
```

Comment:
Do we need to update ``fa`` CDN? If not, then let's keep the old one since this will increase file size download on load, a problem we are already dealing with.

Reply:
In version 5 Font awesome had depreciated `fa` and instead `fas` is used. And the fact they renamed that icon 'hand-o-up' from to `hand-point-up` in version 5. Import that one script is enough.